code/modules/mob/hear_say.dm

```diff
@@ -91,8 +91,6 @@
 
 	var/track = null
 	if(isobserver(src))
-		if(italics && is_preference_enabled(/datum/client_preference/ghost_radio))
```

Comment:
Have you tested this, and does it show ghosts both the radio message and the whispered message like this preference was intended to avoid?

Reply:
I've tested it, and it does show both. I don't consider this a downside or problem.